lib/nucore.rb

```diff
@@ -108,6 +108,10 @@ def self.included(base)
       end
       module ClassMethods
 
+        def order_by_asc_nulls_first(field)
+          NUCore::Database.oracle? ? order("#{field} asc nulls first") : order(field)
+        end
+
         def order_by_desc_nulls_first(field)
           NUCore::Database.oracle? ? order("#{field} desc nulls first") : order("-#{field}")
```

Comment:
I decided not to change this even if the default in oracle for `desc` is nulls first because I'd rather be explicit about it.